client/locations-list-map.js

```diff
@@ -868,8 +894,9 @@ function createRequesterListItemEl(entry) {
   entry.domElem = ce('div', 'location');
   const header = ce('div', 'd-flex');
   const headerHospitalInfo = ce('div', 'flex-grow-1');
+  const headerZoomLink = ce('div', 'icon icon-search entry-zoom-link');
```

Comment:
Can you add an aria-label to the div? Something like "Zoom to marker"?

Reply:
added

Reply:
ship it!